### ISSUE TITLE
Pinch zoom does not work on Android WebView

### ISSUE THREAD

**Issue by mhsmith** [beeware/toga#2258]
*Originally posted by @t-arn on Discord*

Is it possible to use the 2-finger-gesture to zoom a WebView content on Android? Yesterday, I suddenly had this working, but I don't know why it worked then. Because now, it does not work anymore and I didn't really change anything in that area.

It has to do with the screen orientation. Switching several times between portrait and landscape makes the zoom work. But where is the logic? I would like to have the zooming always active...

[...]

I found the solution. The following code enables pinch zoom on Android:
```
webview._impl.native.getSettings().setBuiltInZoomControls(True)
webview._impl.native.getSettings().setDisplayZoomControls(False)
```

I don't know why it sometimes works without this code. 
Should we add this to the Android implementation of WebView?

-----

*Originally posted by @freakboy3742 on Discord*

I haven’t dug into the specifics enough to know if there are other consequences, but on the face of it, it seems like a reasonable change.

**Comment by mhsmith** [comment]
Yes, this looks reasonable to me too.

**Comment by t-arn** [comment]
I will make a PR tomorrow.

**Comment by t-arn** [comment]
@mhsmith We can close this issue since it has been solved with PR #2261

**Comment by mhsmith** [comment]
Thanks; in future, you can link a PR to an issue automatically by adding text like "closes #2258" somewhere in the PR's first comment.
